docs/src/main/asciidoc/openapi-swaggerui-guide.adoc

```diff
@@ -284,6 +284,93 @@ quarkus.smallrye-openapi.path=/swagger
 
 Hit `CTRL+C` to stop the application.
 
+== Loading OpenAPI Schama From Static Files
+
+Instead of dynamically creating OpenAPI schemas from annotation scanning, Quarkus also proposes serving static OpenAPI documents.
+The static file to serve must be a valid document conforming to the https://swagger.io/docs/specification[OpenAPI specification].
+An OpenAPI document that conforms to the  OpenAPI Specification is itself a valid JSON object, that can be represented in
+a `yaml` or `json` formats.
+
+To see this in action, we'll create a static OpenAPI document `META-INF/openapi.yaml` for our `/fruits` path.
+You can choose a different document path as indicating in the <<open-document-paths, document paths tip below>>
+
+[source, shell]
+----
+openapi: 3.0.1
+info:
+  title: Static OpenAPI document of fruits resource
+  description: Fruit resources Open API documentation
+  version: "1.0"
+
+servers:
+  - url: http://localhost:8080/openapi
+    description: Optional dev mode server description
+
+paths:
+  /fruits:
+    get:
+      responses:
+        200:
+          description: OK - fruits list
+          content:
+            application/json: {}
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: new fruit resource created
+          content:
+            application/json: {}
+    delete:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: OK - fruit resource deleted
+          content:
+            application/json: {}
+components:
+  schemas:
+    Fruit:
+      properties:
+        description:
+          type: string
+        name:
+          type: string
+----
+
+Now, a request to `/openapi` endpoint will serve the static OpenAPI document instead of the generated one.
```

Comment:
For this to be true I think scanning needs to be disabled, otherwise it merges the two models

Reply:
Yes, I have added a TIP just below.

Reply:
I'm wondering if it needs to be made explicit here, as currently the example will serve the combined model and not just the OpenAPI static file

Reply:
Good point. So I have remove the tip and added a few sentences based on your suggestion. WDYT?

Reply:
I think that looks good, thanks